src/Infrastructure/Network.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+
+namespace evias\NEMBlockchain\Infrastructure;
+
+use evias\NEMBlockchain\NemSDK;
+
+class Network {
+
+	public $nemSDK;
+	public $version;
+	public $version2;
```

Comment:
instead of having version and version2, i suggest using an array of *versions IDs per network*